integration-tests/deployment/helpers.go

```diff
@@ -13,9 +13,26 @@ import (
 	"github.com/ethereum/go-ethereum/accounts/abi/bind"
 	"github.com/ethereum/go-ethereum/common"
 	"github.com/ethereum/go-ethereum/core/types"
+	"github.com/ethereum/go-ethereum/crypto"
 	"github.com/pkg/errors"
 )
 
+type OCRSecrets struct {
```

Comment:
code comment please about what and why.

i thought that research was going to provide a func in libocr for this, so i'm confused how who is expected to use it and under what conditions

Reply:
added a comment for the struct. See below for the new libocr fn